packages/block-library/src/site-logo/edit.js

```diff
@@ -54,9 +55,37 @@ import { MIN_SIZE } from '../image/constants';
 const ALLOWED_MEDIA_TYPES = [ 'image' ];
 const ACCEPT_MEDIA_STRING = 'image/*';
 
+// If the logo is linked, wrap in an <a /> tag to trigger any inherited link element styles.
+const ImageWrapper = ( { isLink, href, title, classes, children } ) => {
```

Comment:
```suggestion
const ImageWrapper = ( { isLink, href, title, classes, children } ) => {
```

We should probably use the more standard `className` prop for this component.

Reply:
I can't even remember what classes I was trying to add to the ImageWrapper and why.